### PR TITLE
Extract long strings and make available for translation.

### PR DESCRIPTION
**Pull request by brendandburns**
@fabianofranz @zen

### CODE REVIEW

**Comment by fabianofranz** [comment]
@k8s-bot verify test this

File: pkg/kubectl/cmd/create_secret.go [hunk 1]
```diff
@@ -116,25 +116,27 @@ func CreateSecretGeneric(f cmdutil.Factory, cmdOut io.Writer, cmd *cobra.Command
 }
 
 var (
-	secretForDockerRegistryLong = templates.LongDesc(`
-		Create a new secret for use with Docker registries.
+	secretForDockerRegistryLong = templates.LongDesc(i18n.T(`
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/create_secret.go]: @brendandburns looks like the description and examples from generic secrets are being used in the tls and docker registry subcommands here, maybe the script is not safe for files with more than one description and examples?  ;)

File: pkg/kubectl/cmd/create_service.go [hunk 2]
```diff
@@ -110,12 +110,15 @@ func CreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer, cmd *cobra.Comm
 }
 
 var (
-	serviceNodePortLong = templates.LongDesc(`
-    Create a nodeport service with the specified name.`)
+	serviceNodePortLong = templates.LongDesc(i18n.T(`
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/create_service.go]: Same here.

File: pkg/kubectl/cmd/drain.go [hunk 3]
```diff
@@ -112,12 +112,12 @@ func NewCmdCordon(f cmdutil.Factory, out io.Writer) *cobra.Command {
 }
 
 var (
-	uncordon_long = templates.LongDesc(`
-		Mark node as schedulable.`)
+	uncordon_long = templates.LongDesc(i18n.T(`
+		Mark node as unschedulable.`))
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/drain.go]: ...and here.

File: pkg/kubectl/cmd/replace.go [hunk 4]
```diff
@@ -37,28 +37,28 @@ import (
...
+	replace_example = templates.Examples(i18n.T(`
 		# Replace a pod using the data in pod.json.
 		kubectl replace -f ./pod.json
 
 		# Replace a pod based on the JSON passed into stdin.
 		cat pod.json | kubectl replace -f -
 
 		# Update a single-container pod's image version (tag) to v4
-		kubectl get pod mypod -o yaml | sed 's/\(image: myimage\):.*$/\1:v4/' | kubectl replace -f -
+		kubectl get pod mypod -o yaml | sed 's/\(image: myimage\):.*$/Examples(:v4/' | kubectl replace -f -
```
Comment (fabianofranz) [inline pkg/kubectl/cmd/replace.go]: Was this intended?

File: pkg/kubectl/cmd/drain.go [hunk 5]
```diff
@@ -137,37 +137,12 @@ func NewCmdUncordon(f cmdutil.Factory, out io.Writer) *cobra.Command {
...
-
-		![Workflow](http://kubernetes.io/images/docs/kubectl_drain.svg)`)
-
-	drain_example = templates.Examples(`
-		# Drain node "foo", even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.
-		$ kubectl drain foo --force
-
-		# As above, but abort if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet, and use a grace period of 15 minutes.
-		$ kubectl drain foo --grace-period=900`)
+	drain_long = templates.LongDesc(i18n.T(`
```
Comment (soltysh) [inline pkg/kubectl/cmd/drain.go]: I'm not sure we want to shorten this description, and if so please move it to docs and leave a link about that here.

File: pkg/kubectl/cmd/drain.go (hunk 3 above)
Comment (soltysh) [inline pkg/kubectl/cmd/drain.go]: Additionally cordon/unschedulable and uncordon/schedulable is messed up as well.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by brendandburns** [comment]
I'm guessing my script isn't safe. I will retest and fix...

**Comment by brendandburns** [comment]
@fabianofranz @soltysh 

Fixed up my script and rebased.

Please take another look.

**Review by soltysh — APPROVED** [review]

**Comment by fabianofranz** [comment]
@k8s-bot verify test this

**Comment by brendandburns** [comment]
Self-lgtm as the change was just an `update-bazel.sh` run.

**Comment by fabianofranz** [comment]
@k8s-bot gce etcd3 e2e test this

**Pull request merged**